Dokan/Dokan/Scenes/ProductDetails/Views/SliderCollectionCell/SliderCollectionViewCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SliderCollectionViewCell.swift
+//  Dokan
+//
+//  Created by heba isaa on 19/07/2022.
+//
+
+import UIKit
+import Domain
+
+class SliderCollectionViewCell: UICollectionViewCell, NibLoadableView, ReusableView {
+
+    // MARK: - Outlet
+
+    @IBOutlet weak var productPicture: UIImageView!
+    @IBOutlet weak var noOfCurrentPicture: UILabel!
+    @IBOutlet weak var noOfPicture: UILabel!
+
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+    }
+
+ // MARK: - Configurations
+
+    private func configureProductPicture(image: UIImage, currentPictureNumber: Int) {
```

Comment:
If it's a `private` function, how it will be used to configure the `SliderCollectionViewCell`?